tensorboard/plugins/pr_curve/summary_test.py

```diff
@@ -97,5 +97,81 @@ def test1Class(self):
       [[1.0, 1.0, 1.0, 1.0, 0.5, 0.5, 0.5, 0.5, 0.0, 0.0]]
     ], tensor_nd_array.tolist())
 
+  def test3ClassesWithWeight(self):
+    # Generate summaries for showing PR curves in TensorBoard.
```

Comment:
Instead of generating this data, you could also just import your demo and call its `run_all(self.get_temp_dir(), verbose=False)`. It has three classes, after all.